stable/toxiproxy/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+name: toxiproxy
+description:  A TCP proxy to simulate network and system conditions for chaos and resiliency testing.
+version: "1.0"
+appVersion: "2.1.2"
```

Comment:
Can you add the extra fields like from here? https://github.com/deliveryhero/helm-charts/blob/master/stable/cachet/Chart.yaml